src/cockpit/packages.py

```diff
@@ -419,7 +419,10 @@ def serve_manifests_js(self, channel):
 
     def serve_package_file(self, path, channel):
         package, _, package_path = path[1:].partition('/')
-        self.packages[package].serve_file(package_path, channel)
+        try:
+            self.packages[package].serve_file(package_path, channel)
```

Comment:
Two possibilities:
 - there's too much covered by this `try:`: split the dictionary lookup from the function call
 - or: if it's your intention to also handle `KeyError` thrown by `serve_file()` then you could probably also change a use of `.get()` there into `[]` (in `find_find()`) and drop the `found is None` case in `serve_file()` — possibly simply merging those two functions, then.  This would actually be a nice cleanup.